### PR TITLE
Rename usage of master to main

### PR DESCRIPTION
**Pull request by leobalter**
Ref #2699

### CODE REVIEW

File: .github/workflows/update-master-after-main.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Updates the master branch after main
+name: Updates the main branch after main
```
Comment (ljharb) [inline .github/workflows/update-master-after-main.yml:1]:
```suggestion
name: Updates the main branch after master
```
or else i'm confused as to what this workflow will do now

Comment (leobalter) [reply .github/workflows/update-master-after-main.yml]: actually... it's supposed to remain in the original state: the workflow updates the master branch after changes are pushed to main

File: .github/workflows/update-master-after-main.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name: Updates the master branch after main
+name: Updates the main branch after master
```
Comment (leobalter) [inline .github/workflows/update-master-after-main.yml:1]:
```suggestion
name: Updates the master branch after changes in the main branch
```

Comment (leobalter) [reply .github/workflows/update-master-after-main.yml:1]: Lazy level: master. Just used github ui to fix it.

**Comment by rwaldron** [comment]
@syg heads up in case V8 has explicit references to this repo and a "master" branch—you're going to want to update to "main".

**Pull request merged**